### PR TITLE
[ARTS-760] - UI tests around Build versions feature

### PR DESCRIPTION
**Pull request by sameerapurini**
## Description
The PR has automation scenarios written around the "about" feature. The tests check if the services and their associated build number with timestamp is loaded on the screen.
As the timestamp is a dynamic value, only its presence on the screen is checked.

### Changes
- [ARTS-760] - UI tests around "about" feature

### Checklist:

- [ ] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [ ] I have included a short-meaningful title, description and changes for this PR

[ARTS-760]: https://ndph-arts.atlassian.net/browse/ARTS-760

### CODE REVIEW

**Review by sameendph — APPROVED** [review]
approving

File: ui-tests/conf/wdio.conf.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ exports.config = {
         source: true,       // <boolean> hide source uris
         profile: [],        // <string[]> (name) specify the profile to use
         strict: false,      // <boolean> fail if there are any undefined or pending steps
-        tagExpression: '@smoketest',  // <string> (expression) only execute the features or scenarios with tags matching the expression
+        tagExpression: '@arts-48',  // <string> (expression) only execute the features or scenarios with tags matching the expression
```
Comment (KateSant) [inline ui-tests/conf/wdio.conf.js:95]: Why arts-48?

File: ui-tests/conf/wdio.conf.js [hunk 2]
```diff
@@ -25,24 +25,24 @@ exports.config = {
...
+        //     maxInstances: 1,
+        //     browserName: 'firefox'
 
-    },
+        // },
 
-    {
-        maxInstances: 1,
-        browserName: 'safari',
+        // {
```
Comment (KateSant) [inline ui-tests/conf/wdio.conf.js:43]: I am not sure what this config does or why it's no longer needed, but usually better to delete than comment-out?

File: ui-tests/conf/wdio.conf.js (hunk 1 above)
Comment (sameerapurini) [reply ui-tests/conf/wdio.conf.js:95]: sorry my mistake, will revert it now :)

File: ui-tests/conf/wdio.conf.js (hunk 2 above)
Comment (sameerapurini) [reply ui-tests/conf/wdio.conf.js:43]: you are right, will need to revert it, thank you

**Review by mdp0 — APPROVED** [review]

**Pull request merged**